src/pages/login/index.js

```diff
@@ -1,10 +1,12 @@
 /* eslint-disable no-unused-vars */
-import { useCallback, useEffect } from 'react';
+import { useCallback, useEffect, useState } from 'react';
 import { useForm } from 'react-hook-form';
 import { zodResolver } from '@hookform/resolvers/zod';
 import { z } from 'zod';
 import { Link, Redirect, useHistory } from 'react-router-dom';
 import { useDispatch } from 'react-redux';
+import FacebookLogin from 'react-facebook-login';
+
 import { api } from '../../services/api';
 import { useLoginMutation } from '../../services/auth/auth';
```

Comment:
WDYT if you apply [absolute imports](https://create-react-app.dev/docs/importing-a-component/#absolute-imports)? In order to avoid writing `../../` everytime that you require a file in a different directory